caravel/assets/spec/javascripts/explore/components/actions.spec.js

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import shortid from 'shortid';
+import * as actions from '../../../../javascripts/explorev2/actions';
+import { initialState } from '../../../../javascripts/explorev2/store';
+import { exploreReducer } from '../../../../javascripts/explorev2/reducers';
```

Comment:
the import name and file name should be the same according to our style guide: https://github.com/airbnb/javascript#naming--filename-matches-export